### ISSUE TITLE
how to make the form properties multi language ? i could not find how to do it.

### ISSUE THREAD

**Issue by madruga8** [json-editor/json-editor#950]
### General information

- json-editor version:

### Steps to reproduce the behavior

Direct link to example: https://json-editor.github.io/json-editor/

```javascript
{
  "title": "json schema example",
  "type": "object",
  "properties": {
    "example": {
      "type": "string",
      "description": "This is an example schema.",
      "default": "Please edit me."
    }
  }
}
```

**Comment by germanbisurgi** [comment]
at the moment is not implemented but there is a PR for this #887

**Comment by germanbisurgi** [comment]
The feature is now merged. look here: #887
